src/Classes/ListControl.lua

```diff
@@ -300,6 +305,11 @@ function ListClass:Draw(viewPort)
 	end
 end
 
+function ListClass:WipeSelections()
+	-- Simple function to be called from outside the class
+	wipeTable(self.selections)
+end
```

Comment:
Unless there's a specific use case for the function outside of this class, I would just keep using `wipeTable` everywhere.

Reply:
Comes from my OOP days. Don't expect the external procedures to have to know about the internals of a class.